src/components/directory-content.js

```diff
@@ -648,50 +648,49 @@ const DirectoryContent = () => {
                             dataKey: 'selected',
                             label: '',
                             headerRenderer: selectionHeaderRenderer,
-                            maxWidth: 42,
+                            minWidth: '4ex',
```

Comment:
just a question : why using 'ex' here for width property, because 'ex' is relative to the x-height of the current font

Reply:
well i use 'ex' to specify the width based on number of characters
example: '20ex'  that's mean allow me a space can contain 20 characters